app/services/github_service.rb

```diff
@@ -0,0 +1,23 @@
+class GithubService
+
+  def self.repo_info
+    response = Faraday.get "https://api.github.com/repos/LawrenceWhalen/little-esty-shop"
+
+    body = response.body
+    JSON.parse(body, symbolize_names: true)
+  end
+
+  def self.contributors_info
+    response = Faraday.get "https://api.github.com/repos/LawrenceWhalen/little-esty-shop/contributors"
+
+    body = response.body
+    JSON.parse(body, symbolize_names: true)
+  end
+
+  def self.closed_pulls
+    response = Faraday.get "https://api.github.com/repos/LawrenceWhalen/little-esty-shop/pulls?state=closed"
+
+    body = response.body
+    JSON.parse(body, symbolize_names: true)
+  end
+end
```

Comment:
I wonder if there's a world in which we just make a method for all of these and then pass in different links. Doesn't work with our current structure, but there's a lot of repeating code here 🧐

Reply:
Good Idea for a refactor, I might work on that today.